<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiDatasetFit</class>
 <widget class="QMainWindow" name="MultiDatasetFit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi-dataset fitting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
    <item>
     <widget class="QSplitter" name="hSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="browserLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnFit">
            <property name="toolTip">
             <string>Start multi-dataset fitting</string>
            </property>
            <property name="text">
             <string>&amp;Fit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbPlotGuess">
            <property name="text">
             <string>Plot Guess</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="vSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTableWidget" name="dataTable">
           <property name="editTriggers">
            <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
           </property>
           <column>
            <property name="text">
             <string>Workspace</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>WS Index</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Start X</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>End X</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="btnAddWorkspace">
             <property name="toolTip">
              <string>Add one or more spectra from a workspace</string>
             </property>
             <property name="text">
              <string>&amp;Add Workspace</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRemove">
             <property name="toolTip">
              <string>Remove selected spectra</string>
             </property>
             <property name="text">
              <string>&amp;Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btnPrev">
             <property name="toolTip">
              <string>Display previous data set</string>
             </property>
             <property name="text">
              <string>&lt;</string>
             </property>
             <property name="shortcut">
              <string>Alt+,</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbPlotSelector">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Select a data set</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnNext">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Display next data set</string>
             </property>
             <property name="text">
              <string>&gt;</string>
             </property>
             <property name="shortcut">
              <string>Alt+.</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="toolOptions">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="ZoomPage">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="cbShowDataErrors">
               <property name="toolTip">
                <string>Show the error bars on the data curve.</string>
               </property>
               <property name="text">
                <string>Show data errors</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnToFittingRange">
               <property name="toolTip">
                <string>Zoom to current fitting range.</string>
               </property>
               <property name="text">
                <string>To Fitting Range</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="RangePage">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="cbApplyRangeToAll">
               <property name="toolTip">
                <string>Apply selected fitting range to all spectra.</string>
               </property>
               <property name="text">
                <string>Apply to all spectra</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnToVisibleRange">
               <property name="toolTip">
                <string>Set fitting range to visible region.</string>
               </property>
               <property name="text">
                <string>To Visible</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>543</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QwtPlot" name="plot">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="infoBar">
      <property name="text">
       <string>infoBar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header location="global">qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnAddWorkspace</tabstop>
  <tabstop>dataTable</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnPrev</tabstop>
  <tabstop>btnFit</tabstop>
  <tabstop>btnNext</tabstop>
  <tabstop>cbPlotSelector</tabstop>
 </tabstops>
 <resources>
  <include location="../../../icons/CustomInterfacesIcons.qrc"/>
 </resources>
 <connections/>
</ui>
